app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -76,11 +100,39 @@ def get_all_records():
     # If request was not successful, there will be no records field in response
     # Just return what is in cached layer and log an error
     except KeyError as e:
-        body = "Results were not successfully retrieved from Airtable API. " \
-               "Please check connection parameters in config.py and fields in airtable_fields_config.json."
-        request_info = {
-            "url": url,
-            "headers": json.dumps(headers)
-        }
-        send_api_error_slack_notif(body, data, error=e, request_info=request_info, channel='#dev-logging-etl')
-        return request_info
+        _handle_airtable_error(e, data, url)
```

Comment:
again just pass in headers that you define before this try catch statement so you just define it once